cmd/kube-controller-manager/app/plugins.go

```diff
@@ -45,6 +46,22 @@ import (
 	"github.com/golang/glog"
 )
 
+// ProbeAttachableVolumePlugins collects all volume plugins for the attach/
+// detach controller. VolumeConfiguration is used ot get FlexVolumePluginDir
+// which specifies the directory to search for additional third party volume
+// plugins.
+// The list of plugins is manually compiled. This code and the plugin
```

Comment:
Is there any reason not to make an "all_plugins" pkg that has all of the volume plugins listed in one place?  This would return, for example, a Secrets plugin here, but that won't answer as an attachable..